### PR TITLE
Bug: Order properties should not be accessed directly

### PR DESCRIPTION
**Pull request by multinauta**
I changed the way to get the  order properties, using getters. I also added a log to store the response from the server when shipments are created.

woocommerce.shipit.cl test  / WP:5.6 /WC:4.8.0
image

### CODE REVIEW

File: shipit/trunk/readme.txt [hunk 1]
```diff
@@ -55,6 +55,11 @@ Sí, con las acciones en lote, más información en el siguiente link: https://s
 
 == Changelog ==
 
+= 3.0.16 =
+*Corrección método de obtener las propiedades de order.
+*Agregamos un log para guardar la respues del servicio al crear el envío.
```
Comment (Zeaneth) [inline shipit/trunk/readme.txt:60]: `respues`

File: shipit/trunk/src/shipit_debug.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php 
+
+class ShipitDebug {
+
+    static function debug($message)
+    {
+        $date = new DateTime();
+        $timeZone = $date->getTimezone();
+        $current_date = date('d/m/Y H:i:s');
```
Comment (Zeaneth) [inline shipit/trunk/src/shipit_debug.php:9]: Is there a reason for not using camel case for `current_date` variable as well?

Comment (multinauta) [reply shipit/trunk/src/shipit_debug.php:9]: nopes,  i will change it

**Review by mnanom — APPROVED** [review]

**Review by luispascuali — APPROVED** [review]

**Review by Zeaneth — COMMENTED** [review]
What about `bulk_actions.php` file? Shouldn't this also be calling order with the getters?

File: shipit/trunk/shipit.php [hunk 3]
```diff
@@ -139,7 +143,8 @@ function dispatchToShipit($orderId) {
       $insuranceSetting = $core->insurance();
       $sellerSetting = $integration->setting();
       $request = createShipment($company, $skus, $sellerSetting, $insuranceSetting, $order, $destinyId);
-
+      $r = json_decode(json_encode($request), true);
```
Comment (hiro-riveros) [inline shipit/trunk/shipit.php:146]: The variable `$r` mean?

File: shipit/trunk/src/shipit_debug.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php 
+
+class ShipitDebug {
+
+    static function debug($message)
+    {
```
Comment (hiro-riveros) [inline shipit/trunk/src/shipit_debug.php:6]: curly braces at the same line of the `debug` function definition.

**Review by hiro-riveros — CHANGES_REQUESTED** [review]

File: shipit/trunk/shipit.php (hunk 3 above)
Comment (multinauta) [reply shipit/trunk/shipit.php:146]: request to array , i will change it

File: shipit/trunk/src/shipit_debug.php (hunk 4 above)
Comment (multinauta) [reply shipit/trunk/src/shipit_debug.php:6]: i will change it

**Review by hiro-riveros — APPROVED** [review]

**Pull request merged**